langchain4j-pgvector/src/main/java/dev/langchain4j/store/embedding/pgvector/PgVectorEmbeddingStore.java

```diff
@@ -238,6 +241,78 @@ public List<String> addAll(List<Embedding> embeddings, List<TextSegment> embedde
         return ids;
     }
 
+    /**
+     * Removes an embedding from the store based on its unique identifier.
+     *
+     * @param id The unique identifier of the embedding to be removed.
+     */
+    @Override
+    public void remove(String id) {
+        try (Connection connection = getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(String.format(
+                    "DELETE FROM %s WHERE embedding_id = ?", table));
+            statement.setObject(1, UUID.fromString(id));
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * Removes multiple embeddings from the store.
+     *
+     * @param ids A list of unique identifiers of the embeddings to be removed.
+     */
+    @Override
+    public void removeAll(List<String> ids) {
+        if (ids != null && !ids.isEmpty()) {
+            try (Connection connection = getConnection()) {
+                PreparedStatement statement = connection.prepareStatement(String.format(
```

Comment:
PreparedStatement needs closing, so this needs to be wrapped in a try-with-resources too